Sources/PandaSDK/Views/WebViewController.swift

```diff
@@ -85,7 +85,7 @@ final class WebViewController: UIViewController, WKScriptMessageHandler {
         
 
         if let html = html {
-            load(html: html, baseURL: url?.url)
+            load(html: html, baseURL: (url?.url == nil ? URL(fileURLWithPath: Bundle.main.bundlePath) : url?.url))
```

Comment:
Will `URL(fileURLWithPath: Bundle.main.bundlePath)` direct to app or to panda framework?
Do you need `url?.url == nil` check?